egov/egov-pgr/src/main/java/org/egov/pgr/service/ComplaintService.java

```diff
@@ -398,4 +402,48 @@ public void sendEmailandSms(final Complaint complaint) {
         return Arrays.asList(ReceivingMode.values());
     }
 
+    public List<Complaint> getLatest(int page, int pageSize) {
```

Comment:
Please write Data Access Logic in repository ie : querying to DB should goes to corresponding repository